course/layouts/shortcodes/resource.html

```diff
@@ -1,9 +1,16 @@
-{{- $uuid := index .Params 0 -}}
+{{- $uuid := default "" (.Get "uuid") -}}
+{{- $href := default "" (.Get "href") -}}
+{{- $href_uuid := default "" (.Get "href_uuid") -}}
+
```

Comment:
I think it will be better if we get this href and href_uuid in image_resource partial instead of getting them here and passing them as a parameter to image_resource partial as `resource.html` as no usage of them, but I was having an issue while doing that.
@gumaerc Do you have any thoughts on this?

Reply:
@MAbdurrehman12 one of the issues is that `ocw-studio` isn't set up to embed an "`image_resource`" shortcode on its own.  The shortcode would need to be created here in `ocw-hugo-themes`, and then `ocw-studio` would need to be set up to utilize it.  You do have a good point though in that I can't think of another type of embeddable resource that would be wrapped in a link.  What do you think @pdpinch ?

Reply:
FYI, the downside of accepting these parameters on the `resource` shortcode has to do with the fact that we currently only have one positional parameter on it, the UUID.  If we decide we want to start passing in other things like `href` then we will need to also have `uuid` be a named agument in all existing `resource` shortcodes, so we will have to go through and replace that in the existing data.

Reply:
but do let me know if you find out any other issue as we have quite a lot of resources so its better to play safe